### PR TITLE
Just the ci

### PR DESCRIPTION
**Pull request by Donistivanov**
I think these commits contain only the CI

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: python
```
Comment (Byvire) [inline .travis.yml]: You could add a comment clarifying that this file is intended to be used primarily by Travis CI, but I won't force you to.

**Comment by Byvire** [comment]
Do please make sure the CI works before merging to master.

File: .travis.yml (hunk 1 above)
Comment (Donistivanov) [reply .travis.yml]: I decided against doing this because the name of the file has "travis" in it!

**Comment by Donistivanov** [comment]
I am merging this so @Agizin maybe we can turn CI on!

**Comment by Agizin** [comment]
Travis should be reactivated now.

**Pull request merged**